Formula/fluid-synth.rb

```diff
@@ -14,6 +14,7 @@ class FluidSynth < Formula
   depends_on "cmake" => :build
   depends_on "pkg-config" => :build
   depends_on "glib"
+  depends_on "dbus" => :optional
```

Comment:
No new option please. What would be the user benefit of this new dependency?

Reply:
It's to request realtime status from rtkit, but since rtkit seems to only support linux, you're correct. I'll remove it.